src/game_inventory.cpp

```diff
@@ -522,6 +522,23 @@ class comestible_inventory_preset : public inventory_selector_preset
                 }
             }, _( "JOY" ) );
 
+            append_cell( []( const item_location & loc ) {
+                const int health = loc->is_comestible() ? loc->get_comestible()->healthy : 0;
+                if( health > 3 ) {
+                    return _( "<good>+++</good>" );
+                } else if( health > 0 ) {
+                    return _( "<good>+</good>" );
+                } else if( health < -3 ) {
+                    return _( "<bad>!!!</bad>" );
+                } else if( health < 0 ) {
+                    return _( "<bad>-</bad>" );
```

Comment:
I think they no longer need to be translated either
```suggestion
                    return "<good>+++</good>";
                } else if( health > 0 ) {
                    return "<good>+</good>";
                } else if( health < -3 ) {
                    return "<bad>!!!</bad>";
                } else if( health < 0 ) {
                    return "<bad>-</bad>";
```

Reply:
oh yeah I forgot the _() is for translations lol